tests/platform_tests/counterpoll/test_counterpoll_watermark.py

```diff
@@ -84,7 +88,13 @@ def test_counterpoll_queue_watermark_pg_drop(duthosts, localhost, enum_rand_one_
            no WATERMARK or QUEUE stats in FLEX_COUNTER_DB
     4. enables all three counterpolls (queue,watermark,pg-drop) and count stats per type
     """
-    duthost = duthosts[enum_rand_one_per_hwsku_hostname]
+    duthost = duthosts[enum_rand_one_per_hwsku_frontend_hostname]
+
+    if duthost.is_multi_asic:
```

Comment:
I think the `enum_rand_one_frontend_asic_index` will return `None` for single asic. We can remove this check.